postgresql/connection.go

```diff
@@ -0,0 +1,95 @@
+package postgresql
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"os"
+
+	_ "github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/dialers/postgres" // blank import
+	"github.com/joho/godotenv"
+
+	"github.com/benchttp/server/benchttp"
+)
+
+type computedStatsService struct {
+	db *sql.DB
+}
+
+func NewComputedStatsService(idleConn, maxConn int) (benchttp.ComputedStatsService, error) {
+	connectionInfo, err := getConnectionInfoFromEnvVariables()
+	if err != nil {
+		return nil, err
+	}
+
+	dbURI := fmt.Sprintf("host=%s user=%s password=%s dbname=%s sslmode=disable",
+		connectionInfo.host,
+		connectionInfo.user,
+		connectionInfo.password,
+		connectionInfo.dbName)
+
+	db, err := sql.Open("cloudsqlpostgres", dbURI)
+	if err != nil {
+		return nil, ErrDatabaseConnection
+	}
+
+	err = db.Ping()
+	if err != nil {
+		return nil, ErrDatabasePing
+	}
+
+	db.SetMaxIdleConns(idleConn)
+	db.SetMaxOpenConns(maxConn)
+
+	return &computedStatsService{db}, nil
+}
+
+func (b *computedStatsService) Close() {
```

Comment:
I've just realized that I have not used it anywhere :thinking:
When are my connections actually closing at the moment ? :sweat_smile:
Wouldn't it actually be better if I had a `connect()` method in `connection.go` outside `NewComputedStatsService` which would be called in each `endpoint_method` beginning? And a defer `db.Close()` in each of them? :thinking: